wagon-providers/wagon-http-shared/src/main/java/org/apache/maven/wagon/shared/http/AbstractHttpClientWagon.java

```diff
@@ -82,6 +82,7 @@
 import org.apache.maven.wagon.resource.Resource;
 import org.codehaus.plexus.util.StringUtils;
 
+import javax.activation.MimetypesFileTypeMap;
```

Comment:
This class is not significant enough to pull in the javax.activation package. I'm sure there's an alternative somewhere. Or just use `URLConnection.guessContentTypeFromName`

Reply:
fixed in latest push